src/app/views/orders/order.controller.coffee

```diff
@@ -4,8 +4,9 @@
 
   vm.orderId = $stateParams.orderId
   vm.orgId = $stateParams.orgId
-
   vm.order = {}
+  vm.rootName = $filter('translate')('mnoe_admin_panel.dashboard.order.root_name')
+  vm.order.provisioning_data = {'':''}
```

Comment:
@alexnoox I had to init the object this way, because otherwise the directive would not create the user friendly Json tree, instead it would just bind the data the same way it comes from the back end once it is defined.
I tried

vm.order.provisoning_data = JSON.stringify(vm.order.provisoning_data)
and similar approaches, none of them work

Do you have any suggestion?

Thanks a lot

Reply:
Isn't `vm.order.provisioning_data = {}` working?

Reply:
No, it does not I am afraid. I find it quite ugly, but I can not find a better way. What do you reckon?

Reply:
Have you tried initializing the `vm.order` object only when fetching the subscription? Maybe add a `ng-if` on the selector to check that there is `vm.order.provisoning_data`?

Reply:
Thanks a lot, @alexnoox, works like a charm.